pkg/render/kube-controllers.go

```diff
@@ -259,6 +269,33 @@ func (c *kubeControllersComponent) controllersDeployment() *apps.Deployment {
 									},
 								},
 							},
+							VolumeMounts: []v1.VolumeMount{{
+								Name:      "manager-cert",
+								MountPath: "/manager-tls",
+								ReadOnly:  true,
+							}},
+						},
+					},
+					Volumes: []v1.Volume{
+						{
+							Name: "manager-cert",
+							VolumeSource: v1.VolumeSource{
+								Secret: &v1.SecretVolumeSource{
+									DefaultMode: &defaultMode,
+									SecretName:  ManagerTLSSecretName,
+									Optional:   &optional,
```

Comment:
Same as in the compliance, we know if the secret exists or not so there should be no need for this to be an optional mount.